src/compiler.js

```diff
@@ -95,8 +95,8 @@ class CompiledProtodef {
     return buffer
   }
 
-  parsePacketBuffer (type, buffer) {
-    const { value, size } = tryCatch(() => this.read(buffer, 0, type),
+  parsePacketBuffer (type, buffer, offset = 0) {
```

Comment:
can you update the doc for this change in API ?

Reply:
done